packages/assets/lib/upload.js

```diff
@@ -30,13 +43,19 @@ const uploadPortrait = async (portrait) => {
 
   const inputBuffer = Buffer.from(portrait, 'base64')
 
-  const {meta, data} = await convertImage.toJPEG(inputBuffer)
-  await uploadS3({
-    stream: data,
-    path: portraitPath,
-    mimeType: 'image/jpeg',
-    bucket: AWS_S3_BUCKET
-  })
+  const { meta, data } = isJPEG
```

Comment:
Ok, maybe:

```
  const { meta, data } = await toJPEG(inputBuffer)
```

And have in `toJPEG` await metadata (which is done either way) and circumvent rotate/jpeg pipeline. Whatever buffer you shovel in, it will be a JPEG 100%. Neat and tight.

And if you like that, then I am thinking that `uploadPortrait` should not trust `isJPEG` argument either way and would drop it.